README.md

```
@@ -9,41 +9,40 @@
+    - [Legacy releases (CVMFS + aliBuild, ≤26.04)](#legacy-releases-cvmfs--alibuild-2604)
 
 <!-- markdown-toc end -->
 
 ## Introduction
 
 FairShip is the software framework for the SHiP experiment which is based on
-FairRoot. The dependencies of FairShip are tracked and installed using
-[alibuild](https://alisw.github.io/alibuild/). A wiki can be found [here](https://github.com/ShipSoft/FairShip/wiki)
+FairRoot. Its dependencies are managed with [pixi](https://pixi.sh) — see
+[Using pixi](#using-pixi) below. A wiki can be found
+[here](https://github.com/ShipSoft/FairShip/wiki).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)